### PR TITLE
Fix #8972: Master server unable to query game info.

### LINKED ISSUE #8972: Servers running network version 0.2.2-9 do not show up on master server list.
**OS:** [Server2012]
**Version:** [0.2.2-9]
**Commit/Build:** [d41822e]

Servers running network version 0.2.2-9 do not show up on master server list.
users can only connect if they know the servers IP address and port number, or if they previously marked the server as favorite .

- [ ] Reproducible in RCT2 (vanilla)?
- [X] Specific to multiplayer?

**Steps to reproduce:**
1.  Host a server running the latest build.
2.  Check server list. It wont be there

### PR DESCRIPTION
**Pull request by ZehMatt**
My bad, didn't think it would matter.

### CODE REVIEW

File: src/openrct2/network/NetworkTypes.h [hunk 1]
```diff
@@ -59,7 +59,7 @@ enum NETWORK_COMMAND
     NETWORK_COMMAND_PING,
     NETWORK_COMMAND_PINGLIST,
     NETWORK_COMMAND_SETDISCONNECTMSG,
-    NETWORK_COMMAND_GAMEINFO,
+    NETWORK_COMMAND_GAMEINFO = 9, // Must always be 9 for the master server.
```
Comment (AaronVanGeffen) [inline src/openrct2/network/NetworkTypes.h]: Wouldn't this make `NETWORK_COMMAND_GAMEINFO` = `NETWORK_COMMAND_SETDISCONNECTMSG` ?

Comment (ZehMatt) [reply src/openrct2/network/NetworkTypes.h]: Right, we'll ill just move the new one down then.

**Comment by duncanspumpkin** [comment]
Lol

**Comment by Gymnasiast** [comment]
Could you rebase this? The network version has changed in the meantime.

**Comment by ZehMatt** [comment]
Done

**Pull request merged**